Assets/AppCenter/Plugins/AppCenterSDK/Crashes/Shared/Crashes.cs

```diff
@@ -47,11 +47,15 @@ public static void AddNativeType(List<RawType> nativeTypes)
 
         public static void TrackError(Exception exception, IDictionary<string, string> properties = null, params ErrorAttachmentLog[] attachments)
         {
-            if (exception != null)
+            if (exception != null && _reportUnhandledExceptions)
             {
                 var exceptionWrapper = CreateWrapperException(exception);
                 CrashesInternal.TrackException(exceptionWrapper.GetRawObject(), properties, attachments);
             }
+            else
+            {
+                Debug.LogWarning("Cannot sending track exception without enabling unhandled exception reporting.");
```

Comment:
We log this warning if `exception` is null and `_reportUnhandledExceptions` is true. I think it could be confusing to get a warning saying about unhandled exception reporting disabled when it is enabled.